udata/tests/test_model.py

```diff
@@ -261,14 +261,14 @@ def test_handle_unicode(self):
         obj.save().reload()
         self.assertEqual(obj.url, url)
 
-    def test_public(self):
+    def test_public_private(self):
         url = 'http://10.10.0.2/path/'
-        URLTester(url=url).save()
+        PrivateURLTester(url=url).save()
         with self.assertRaises(ValidationError):
-            PublicURLTester(url=url).save()
+            URLTester(url=url).save()
 
 
-class DatetimedTest(DBTestMixin, TestCase):
+class Darivate(DBTestMixin, TestCase):
```

Comment:
`Darivate`?

Reply:
Weird, I do known from where that comes from

Reply:
Probably a multi-select gone wrong ;-)

Reply:
Yes, it must be